src/caselawclient/models/utilities/__init__.py

```diff
@@ -3,23 +3,22 @@
 
 from requests_toolbelt.multipart.decoder import BodyPart
 
-VERSION_REGEX = r"xml_versions/(\d{1,10})-(\d{1,10}|TDR)"
+from caselawclient.types import DocumentURIString, MarkLogicDocumentURIString
+
+VERSION_REGEX = r"xml_versions/(\d{1,10})-"
```

Comment:
Since we've dropped the document reference bit of the regex, we probably want to drop that from the explanation below.